app/components/preprint-form-project-select.js

```diff
@@ -5,84 +5,57 @@ import { inject as service } from '@ember/service';
  */
 
 /**
- * Preprint form project select widget - handles all ADD mode cases where the first step is to
- * select an existing OSF project to contain your preprint.  Also used in EDIT mode - as we
- * keep the project locked after preprint has been published.
- * Therefore, you must use an existing project!
+ * Preprint form project select widget - used wherever you need to load a list of user projects
+ * Used in two places - where the user is selecting a file from an existing project
+ * (to copy to their preprint), and where a user is choosing their supplemental project
  *
- *  Uses the file-uploader component, hence the large number of properties for this component,
- *  that are passed along to the file-uploader. Cases not needing the file-uploader are where
- *  you are selecting an existing file on an existing node, or copying a file into
- *  a newly-created component - no file uploading needed.
  *
  * {{preprint-form-project-select
  * ```handlebars
  *      changeInitialState=(action 'changeInitialState')
- *      finishUpload=(action 'finishUpload')
  *     clearDownstreamFields=(action 'clearDownstreamFields')
  *     nextUploadSection=(action 'nextUploadSection')
- *     existingNodeExistingFile=(action 'existingNodeExistingFile')
- *     createComponentCopyFile=(action "createComponentCopyFile")
  *     selectFile=(action "selectExistingFile")
- *     highlightSuccessOrFailure=(action 'highlightSuccessOrFailure')
- *     startPreprint=(action 'startPreprint')
  *     discardUploadChanges=(action 'discardUploadChanges')
  *     startState=_State.START
- *     existingState=existingState
- *     _existingState=_existingState
+ *     uploadInProgress=uploadInProgress
  *     title=title
  *     currentUser=user
  *     selectedFile=selectedFile
  *     hasFile=hasFile
- *     file=file
- *     node=node
- *     userNodes=userNodes
  *     selectedNode=node
- *     contributors=contributors
- *     fileSelect=true
- *     currentState=filePickerState
- *     parentNode=parentNode
- *     convertProjectConfirmed=convertProjectConfirmed
- *     userNodesLoaded=userNodesLoaded
- *     convertOrCopy=convertOrCopy
- *     isTopLevelNode=isTopLevelNode
- *     nodeLocked=nodeLocked
+ *     currentState=currentState
+ *     preprintLocked=preprintLocked
  *     osfStorageProvider=osfStorageProvider
  *     osfProviderLoaded=osfProviderLoaded
  *     titleValid=titleValid
- *     uploadChanged=uploadChanged
- *     uploadInProgress=uploadInProgress
- *     abandonedPreprint=abandonedPreprint
- *     resumeAbandonedPreprint=(action 'resumeAbandonedPreprint')
  *     basicsAbstract=basicsAbstract
  *     editMode=editMode
- *     newNode=newNode
- *     applyLicense=applyLicense
+ *     getProjectContributors=(action 'getProjectContributors')
+ *     provider=currentProvider
+ *     createPreprintCopyFile=(action 'createPreprintCopyFile')
  * }}
  * @class preprint-form-project-select
  */
 export default Component.extend(Analytics, {
```

Comment:
The preprint-form-project-select component is used in two places: where the user is electing to select a preprint file from an existing project, and where the user is selecting a supplemental project.